orderer/common/channelparticipation/restapi.go

```diff
@@ -70,13 +70,84 @@ func NewHTTPHandler(config localconfig.ChannelParticipation, registrar ChannelMa
 		router:    mux.NewRouter(),
 	}
 
+	// swagger:operation GET /channels/{channelID} channels listChannel
+	// ---
+	// summary: Returns detailed channel information for an Ordering Service Node (OSN).
+	// parameters:
+	// - name: channelID
+	//   in: path
+	//   description: Channel ID
+	//   required: true
+	//   type: string
+	// responses:
+	//    "200":
+	//       description: Successfully retrieved channel.
+	//       schema:
+	//         "$ref": "#/definitions/channelInfo"
+
 	handler.router.HandleFunc(urlWithChannelIDKey, handler.serveListOne).Methods(http.MethodGet)
 
+	// swagger:operation DELETE /channels/{channelID} channels deleteChannel
+	// ---
+	// summary: The OSN is removed from the channel via a config transaction.
```

Comment:
I think this should mirror the Join summary, especially since this isn't done via a config transaction.
```suggestion
	// summary: Removes an Ordering Service Node (OSN) from a channel.

Reply:
🧹